scout/server/blueprints/panels/templates/panels/panels.html

```diff
@@ -27,6 +28,48 @@
   </div>
 {% endblock %} <!-- end of content_main -->
 
+
+
+ 
+<!-- Create a Bootstrap card containing text bar and submit button for searching -->
+{% macro search_gene_card() %}
+<div class="container-float">
+  <div class="card mt-3">
+    <div class="card-body">
+      <form action="{{ url_for('panels.panels') }}" enctype="multipart/form-data" method="POST">
+        <div class="row">
+          <div class="col-5">
+            <input type="text" name="search_for" class="form-control" placeholder="Search Gene in Panels" required
```

Comment:
Have you checked how the gene search is performed on other pages, for instance on the genes page?

The search there is more user friendly than a verbatim search because when you enter 3 letters or more it suggests you the possible genes, by matching the entered letters in gene names and also aliases:

<img width="859" alt="image" src="https://user-images.githubusercontent.com/28093618/183578770-327bc0f1-aeca-491f-9e6c-d8227de2aa65.png">

What happens in your code:

<img width="745" alt="image" src="https://user-images.githubusercontent.com/28093618/183578836-00692dde-3527-4e39-b414-66169354cd5e.png">

But you don't know if the gene is not found because you spelled it wrong or because it changed official name and it became an alias or because it doesn't exist in Scout.

Reply:
Typeahead is copied from genes-page.